src/virtualenv/create/via_global_ref/builtin/pypy/pypy3.py

```diff
@@ -20,11 +20,6 @@ def exe_names(cls, interpreter):
 class PyPy3Posix(PyPy3, PosixSupports):
     """PyPy 3 on POSIX"""
 
-    @property
-    def stdlib(self):
-        """PyPy3 respects sysconfig only for the host python, virtual envs is instead lib/pythonx.y/site-packages"""
```

Comment:
This comment was the impetus for the PR since it is clearly wrong, it should say `lib/pypyx.y/site-packages`